wrapanapi/vcloud.py

```diff
@@ -0,0 +1,66 @@
+# coding: utf-8
+from pyvcloud.vcd.client import Client, BasicLoginCredentials
+from pyvcloud.vcd.org import Org
+from pyvcloud.vcd.vdc import VDC
+from pyvcloud.vcd.vapp import VApp
+
+from base import WrapanapiAPIBase
+
+
+class VmwareCloudSystem(WrapanapiAPIBase):
+    """Client to VMware vCloud API"""
+
+    def __init__(self, hostname, username, organization, password, api_port, api_version, **kwargs):
+        self.endpoint = 'https://{}:{}'.format(hostname, api_port)
+        self.username = username
+        self.organization = organization
+        self.password = password
+        self.api_version = api_version
+        super(VmwareCloudSystem, self).__init__(kwargs)
+
+    def stats(self, *requested_stats):
```

Comment:
**REQUIRED**: As in other providers, is it possible to have a persistent "session/client" instance that you can keep reusing, rather than instantiating it in the stats method?

Reply:
You're totally right, thanks for noticing.